app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/EnterCommoditiesDetailsController.scala

```diff
@@ -56,173 +56,66 @@ class EnterCommoditiesDetailsController @Inject() (
     f: (
       SessionData,
       FillingOutClaim,
-      CommoditiesDetailsAnswer
+      Option[CommodityDetails]
```

Comment:
This should be re-named because the page is concerned with reasons for claims not the commodity details.

Reply:
How it should be named? should it be ReasonsForClaims?

Reply:
Can you review this Mikail? I need it to demonstrate my solution soon

Reply:
I don't mind what you call it as long as it conveys the correct semantics. However, we have a clash now because there is already a model called `ReasonForClaim` which is the answer to the question "Select Reason for Claim" - something does not seem right here as it now looks like we are asking for the same information in two pages? I think we should go back to the team and clarify this.

Reply:
Maybe we could change the existing `ReasonForClaim` to `ReasonForClaimOption` and then use `ReasonForClaim` for this page?